recipe/activate-gcc.sh

```diff
@@ -136,6 +136,7 @@ fi
 
 _CMAKE_ARGS="-DCMAKE_AR=${CONDA_PREFIX}/bin/@CHOST@-ar -DCMAKE_RANLIB=${CONDA_PREFIX}/bin/@CHOST@-ranlib"
 _CMAKE_ARGS="-DCMAKE_LINKER=${CONDA_PREFIX}/bin/@CHOST@-ld -DCMAKE_STRIP=${CONDA_PREFIX}/bin/@CHOST@-strip"
```

Comment:
```suggestion
_CMAKE_ARGS="${_CMAKE_ARGS} -DCMAKE_LINKER=${CONDA_PREFIX}/bin/@CHOST@-ld -DCMAKE_STRIP=${CONDA_PREFIX}/bin/@CHOST@-strip"
```
This looks like the longstanding but that prompts to sometimes define CMAKE_AR on feedstocks.

Reply:
Nice catch. Thank you.